ansible/create_cloud_admin_trust.yml

```diff
@@ -0,0 +1,10 @@
+--- 
+
+- name: Create cloud admin trust
+  shell: |
+    . /root/cloudrc && workloadmgr trust-create --is_cloud_trust True admin --insecure
```

Comment:
Who is creating this /root/cloudrc file?

Reply:
Ansible scripts only